src/openforms/forms/tests/test_import_export.py

```diff
@@ -1401,44 +1401,117 @@ class ImportObjectsAPITests(TempdirMixin, OFVCRMixin, TestCase):
+        resources = {
+            "forms": [
+                {
+                    "active": True,
+                    "name": "Test Form 1",
+                    "internal_name": "Test Form Internal 1",
+                    "slug": "objects-api-no-group",
+                    "uuid": "324cadce-a627-4e3f-b117-37ca232f16b2",
+                    "registration_backends": [
+                        {
+                            "key": "test-backend",
+                            "name": "Test backend",
+                            "backend": "objects_api",
+                            "options": {
+                                "version": 1,
+                                "objecttype": "http://localhost:8001/api/v2/objecttypes/8e46e0a5-b1b4-449b-b9e9-fa3cea655f48",
+                                "objecttype_version": 1,
+                            },
+                        }
+                    ],
+                }
+            ]
+        }
 
-        self.objects_api_group = ObjectsAPIGroupConfig.objects.create(
-            objecttypes_service=ServiceFactory.create(
-                api_root="http://localhost:8001/api/v2/",
-                api_type=APITypes.orc,
-                oas="https://example.com/",
-                header_key="Authorization",
-                # See the docker compose fixtures:
-                header_value="Token 171be5abaf41e7856b423ad513df1ef8f867ff48",
-                auth_type=AuthTypes.api_key,
-            ),
-            objects_service=ServiceFactory.create(
-                api_root="http://localhost:8002/api/v2/",
-                api_type=APITypes.orc,
-                oas="https://example.com/",
-                header_key="Authorization",
-                # See the docker compose fixtures:
-                header_value="Token 7657474c3d75f56ae0abd0d1bf7994b09964dca9",
-                auth_type=AuthTypes.api_key,
-            ),
-            drc_service=ServiceFactory.create(
-                api_root="http://localhost:8003/documenten/api/v1/",
-                api_type=APITypes.drc,
-                # See the docker compose fixtures:
-                client_id="test_client_id",
-                secret="test_secret_key",
-                auth_type=AuthTypes.zgw,
-            ),
-            catalogi_service=ServiceFactory.create(
-                api_root="http://localhost:8003/catalogi/api/v1/",
-                api_type=APITypes.ztc,
-                # See the docker compose fixtures:
-                client_id="test_client_id",
-                secret="test_secret_key",
-                auth_type=AuthTypes.zgw,
-            ),
+        with zipfile.ZipFile(self.filepath, "w") as zip_file:
+            for name, data in resources.items():
+                zip_file.writestr(f"{name}.json", json.dumps(data))
+
+        with self.assertRaises(CommandError):
+            call_command("import", import_file=self.filepath)
```

Comment:
I would like some more assertions about what exactly the error is, this can happen for a dozen different reasons. The only assertion expressed here now is "importing does not work", which is dangerously close to reported bug in the first place.